pkg/sqlite/migrations/011_substitutes_for.go

```diff
@@ -0,0 +1,55 @@
+package migrations
+
+import (
+	"context"
+	"database/sql"
+)
+
+const SubstitutesForMigrationKey = 11
+
+// Register this migration
+func init() {
+	registerMigration(SubstitutesForMigrationKey, substitutesForPropertyMigration)
+}
+
+var substitutesForPropertyMigration = &Migration{
+	Id: SubstitutesForMigrationKey,
+	Up: func(ctx context.Context, tx *sql.Tx) error {
+		sql := `
+		ALTER TABLE operatorbundle 
+		ADD COLUMN substitutesfor TEXT;
+		`
+		_, err := tx.ExecContext(ctx, sql)
+		return err
+	},
+	Down: func(ctx context.Context, tx *sql.Tx) error {
+		foreignKeyOff := `PRAGMA foreign_keys = 0`
+		createTempTable := `CREATE TABLE operatorbundle_backup (name TEXT, csv TEXT, bundle TEXT, bundlepath TEXT, version TEXT, skiprange TEXT, replaces TEXT, skips TEXT)`
```

Comment:
Is this dropping the foreign keys from the table entirely?

Reply:
oh wow - yes it is... good catch!